api/src/home/analysis-platform/data/WorkflowDS/Blueprints/Task.json

```diff
@@ -43,6 +43,22 @@
     }
   ],
   "uiRecipes": [
+    {
+      "name": "Form",
+      "type": "system/SIMOS/UiRecipe",
+      "plugin": "form",
+      "category": "edit",
+      "config": {
+        "type": "DMT-Internal/DMT/ui-plugins/form/FormInput",
+        "attributes": [
+          {
+            "type": "DMT-Internal/DMT/ui-plugins/form/fields/StringField",
+            "name": "outputType",
+            "widget": "BlueprintPickerWidget"
+          }
+        ]
+      }
+    },
```

Comment:
Should remove "Form" from the Blueprints where we have custom plugins. And set "Form" as the default edit plugin for everything instead. Which we override in cases like Task.json

Reply:
true, will remove